### PR TITLE
Fixed some chinese translation

### PR DESCRIPTION
**Pull request by oscnet**
not finished yet , will continue to work.

### CODE REVIEW

**Comment by leelinliew** [comment]
Hi Eric, please refer to line 32, "contribution"

File: src/Translation/locales/zh.json [hunk 1]
```diff
@@ -3,40 +3,40 @@
...
+    "not_enough": "你没有足够用于支付的 ETH",
+    "pay_with": "需支付",
+    "powered_by": "由 {bancor_link} 提供",
+    "rate": "兑换率",
+    "receive": "你将收到",
+    "slippage": "误差",
+    "title": "使用 ETH 兑换 MANA"
   },
   "color_key": {
     "contribution": "小区份额",
```
Comment (ericschallock) [inline src/Translation/locales/zh.json]: @leelinliew @oscnet This is referring to parcels of LAND that the user has "contributed" or donated/given to a community district. The community districts are all made up of LAND parcels that various users have grouped together. When you look at the Atlas, any parcels that you have "contributed" to a district are highlighted in a different color. Let me know if this helps to explain it.

**Comment by oscnet** [comment]
Some English words cannot be found in the zh.json file. 
ex: https://market.decentraland.org/30/-14/detail

File: src/Translation/locales/zh.json (hunk 1 above)
Comment (oscnet) [reply src/Translation/locales/zh.json]: I see, but  I can't find any such contributed parcels at the atlas even if  I have some district parcels. it's all colored as "district" , Is this normal?

**Comment by abarmat** [comment]
Is this finished?

**Pull request closed without merging**